### DISCUSSION TITLE
Next 13 api route handler status code

### DISCUSSION THREAD

**Discussion by alireza-constantin** [vercel/next.js#48397] [Help] [answered]
### Summary

How can i set status code  in new NextResponse  and send it with json

**Comment by wwwdepot** [comment, marked as answer]
The 2nd param is options, you can set status there.

```js
return new NextResponse.json(data, { status: 201 })
```

**Reply by Nezo96** [reply]
Now you saved me from headache. Would be good if this was added to beta documentation because there is nothing about second parameter just:

`Produce a response with the given JSON body.`
`return NextResponse.json({ hello: 'Next.js' })`

**Reply by guilhermeavanci** [reply]
Looks like we can't pass other stuff like `statusText`, is that correct?
